.github/workflows/lint.yml

```
@@ -14,13 +14,23 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
 
-      - name: Install Dependencies
-        run: |
-          python -m pip install --upgrade pip
-          python -m pip install black==20.8b1
+      - name: Install black
+        run: pip install black==21.4b2
 
-      - name: Run Black
-        run: |
-          black --check --diff .
+      - name: Run black
+        run: black --diff .
```

```suggestion
        run: black --check --diff .
```

From my testing, it looks like running `black` with `--diff` will exit with 0 regardless if the file has been requires reformatting. Since we want Github Actions to [display a failure when changes are required](https://docs.github.com/en/actions/creating-actions/setting-exit-codes-for-actions), we still might need to include the [`--check`](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html?#exit-code) flag here